### PR TITLE
Issue 188: Main Table and CRUD Handler Creation for FAQ

### LINKED ISSUE #188: Main Table and CRUD Handler Creation for FAQ

### PR DESCRIPTION
**Pull request by rko7**

### CODE REVIEW

File: models/faq.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const faqSchema = new Schema(
+  {
+    faqCategoryName: {
```
Comment (yuanLeeMidori) [inline models/faq.js]: If we have categories for faqs, I think faqCategory should have its collection, and be linked to faq collection.

**Review by yuanLeeMidori — CHANGES_REQUESTED** [review]
If we have categories for faqs, I think faqCategory should have its collection, and be linked to faq collection.

**Comment by rko7** [comment]
added faqCategory collection

**Comment by yuanLeeMidori** [comment]
@rko7 I think you have some changes in this branch that haven't pushed (e.g. the faqCategoryName). I'll wait till you're push more commit, then review.

**Comment by rko7** [comment]
changed collection field names

**Review by eunbeek — APPROVED** [review]

**Review by yuanLeeMidori — APPROVED** [review]

**Pull request merged**